app.py

```diff
@@ -61,11 +64,20 @@
 
 
 class HieroExport(Application):
+    _BASE_HOOKS = base_hooks
```

Comment:
missing docstring. Please comment what this constant is for. (why is it needed?)